File: CiscoPCCE.Toolkit/Bean/Ciphers.cs
using System.Xml.Serialization;
using System.Collections.Generic;

namespace CiscoPCCE.Toolkit.Bean
{





public class Ciphers : BaseApiBean {
  private List<string?>? cipherlist;

  [XmlElement("cipher")]
  public List<string?>? Cipherlist
  {
      get => cipherlist;
      set => cipherlist = value;
  }


  public void setCipherlist(List<String> cipherlist) {
     this.cipherlist = cipherlist;
  }


}

}

File: JavaSrc/toolkit/bean/Ciphers.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


public class Ciphers extends BaseApiBean {
  private List<String> cipherlist;

  @XmlElement(name="cipher")
  public List<String> getCipherlist() {
     return this.cipherlist;
  }

  public void setCipherlist(List<String> cipherlist) {
     this.cipherlist = cipherlist;
  }


}
